h/services/list_organizations.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+from h import models
+
+
+class ListOrganizationsService(object):
+
+    """
+    A service for providing a list of organizations.
+    """
+
+    def __init__(self, session, authority):
+        """
+        Create a new list_organizations service.
+
+        :param _session: the SQLAlchemy session object
+        :param _authority: the authority to use as a default
+        """
+        self._session = session
+        self._authority = authority
+
+    def organizations(self, authority=None):
```

Comment:
At some point, we'll need to figure out what security/access looks like for organizations if we ever expose them as first-class resources. But for now this seems fine...

Reply:
Agree!